notifique/src/main/res/values/colors.xml

```diff
@@ -3,5 +3,6 @@
   <color name="primary">#794C13</color>
   <color name="accent">#A3581E</color>
   <color name="toolbar">@color/primary</color>
+  <color name="scrollbar_background">#cccccc</color>
```

Comment:
nit: uppercase, to match the style of the others.

Reply:
fixed